cray_product_catalog/migration/main.py

```diff
@@ -44,8 +44,31 @@
 LOGGER = logging.getLogger(__name__)
 
 
+# function to check if configmap is already migrated
+def is_migrated():
```

Comment:
This should be a function docstring, not a comment above the function
```suggestion
def is_migrated():
   """
   Check if ConfigMap is already migrated.
   Returns True if so, False if not.
   """
```